test/e2e/json-rpc/wallet_revokePermissions.spec.js

```diff
@@ -0,0 +1,61 @@
+const { strict: assert } = require('assert');
+
+const {
+  withFixtures,
+  defaultGanacheOptions,
+  unlockWallet,
+  openDapp,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Revoke Dapp Permissions', function () {
+  it('should revoke dapp permissions ', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await unlockWallet(driver);
+
+        await openDapp(driver);
+
+        let accountsDiv = await driver.findElement('#accounts');
+
+        assert.equal(
+          await accountsDiv.getText(),
+          '0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
+        );
+
+        // wallet_revokePermissions request
+        const revokePermissionsRequest = JSON.stringify({
+          jsonrpc: '2.0',
+          method: 'wallet_revokePermissions',
+          params: [
+            {
+              eth_accounts: {},
+            },
+          ],
+        });
+
+        const result = await driver.executeScript(
+          `return window.ethereum.request(${revokePermissionsRequest})`,
+        );
+
+        // Response of method call
+        assert.deepEqual(result, null);
+
+        // TODO: Fix having to reload dapp as it is not the proper behavior in production, issue with test setup.
+        await driver.executeScript(`window.location.reload()`);
+        accountsDiv = await driver.findElement('#accounts');
+
+        assert.equal(await accountsDiv.getText(), '');
```

Comment:
Same problem again
```suggestion
        let accountsDiv = await driver.findElement({
          text: '',
          css: '#accounts',
        });
```

Reply:
@HowardBraham This change failed in testing, refer to https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/65354/workflows/64b8aca1-95a7-4030-be9b-312c11542444/jobs/2114759. 
```
<failure message="The locator &apos;[object Object]&apos; is not supported by the E2E test driver" type="Error"><![CDATA[Error: The locator '[object Object]' is not supported by the E2E test driver
```
How should I proceed?

Reply:
Oh sorry about that!  Turns out that you can't use `driver.findElement()` with an empty string, so I wrote it with xpath instead, and it's passing CI now.